src/Parsing/Parser.hs

```diff
@@ -67,37 +62,6 @@ import           TreeSitter.TSX
 import           TreeSitter.TypeScript
 import           TreeSitter.Unmarshal
 
-
--- | A parser, suitable for program analysis, for some specific language, producing 'Term's whose syntax satisfies a list of typeclass constraints.
-data SomeAnalysisParser (constraint :: (* -> *) -> Constraint) ann where
-  SomeAnalysisParser :: ( constraint (Sum fs)
-                        , Apply (VertexDeclaration' (Sum fs)) fs
-                        , HasPrelude lang
-                        )
-                     => Parser (Term (Sum fs) ann)
-                     -> Proxy lang
-                     -> SomeAnalysisParser constraint ann
```

Comment:
Since `VertexDeclaration` can now be applied directly against the term type, we can use `SomeParser` instead.